src/js/other-websites/add_input_icon.js

```diff
@@ -20,30 +22,19 @@ function addRelayMenuToPage(relayMenuWrapper, relayInPageMenu, relayIconBtn) {
   return;
 }
 
-function preventDefaultBehavior(clickEvt) {
-  clickEvt.stopPropagation();
-  clickEvt.stopImmediatePropagation();
-  clickEvt.preventDefault();
-  return;
-}
-
-function getRelayMenuEl() {
-  return document.querySelector(".fx-relay-menu");
-}
-
 function positionRelayMenu() {
-  const relayInPageMenu = getRelayMenuEl();
+  const relayInPageMenu = document.querySelector(".fx-relay-menu-iframe");
   const relayIconBtn = document.querySelector(".fx-relay-menu-open");
   const newIconPosition = relayIconBtn.getBoundingClientRect();
   relayInPageMenu.style.left = newIconPosition.x - 255 + "px";
   relayInPageMenu.style.top = newIconPosition.top + 40 + "px";
 }
 
-let activeElemIndex = -1;
+// let activeElemIndex = -1;
```

Comment:
There are lots of commented-out lines in this file. I'm sure you can find and remove them all yourself :)

Reply:
Cleaned up in 4344dc8e29ad93376f4af2c58eebb7a81182ed08. 🧹